app/ecolabel-translator/src/App.js

```diff
@@ -40,6 +73,27 @@ function App() {
     console.log('Chat ended.');
   };
 
+  const handleChatActive = async () => {
+    setIsChatActive(true);
+    try {
+      const response = await fetch(`http://127.0.0.1:5000/chat/init?label=true`, {
```

Comment:
Variable, and also why ```?label=true```? chat/init label is the name of the label chosen by the user

Reply:
It's named label because that is what chat_init on the backend gets from the request.

Reply:
I'm concerned about the ```=true``` bit